File: src/test/java/test/GetRoleAuthorizedResourcesTest.java
package test
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class GetRoleAuthorizedResourcesTest {

    private static final String ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "YOUR_ACCESS_KEY_SECRET";


    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions(ACCESS_KEY_ID, ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        GetRoleAuthorizedResourcesDto request = new GetRoleAuthorizedResourcesDto();
        request.setCode("code_9900");
        request.setNamespace("namespace_7571");
        request.setResourceType("resourceType_6395");

        RoleAuthorizedResourcePaginatedRespDto response = client.getRoleAuthorizedResources(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/GetRoleAuthorizedResourcesTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.GetRoleAuthorizedResourcesDto{
        Code: "code_8672",
        Namespace: "namespace_4739",
        ResourceType: "resourceType_6300",
    }
    response := client.GetRoleAuthorizedResources(&request)
    fmt.Println(response)
}

